code/game/machinery/newscaster.dm

```diff
@@ -188,11 +186,12 @@ var/list/obj/machinery/newscaster/allCasters = list()
 	var/obj/item/weapon/photo/photo = null
 	var/channel_name = ""
 	var/c_locked=0
-	var/hitstaken = 0
+	var/health = 60
 	var/datum/newscaster/feed_channel/viewing_channel = null
 	var/allow_comments = 1
 	luminosity = 0
 	anchored = 1
+	var/hitstaken = 0 //phil235 to be removed, it's used in a map var edit.
```

Comment:
can't remove it because it's used on a map, and I don't want and don't know how to edit maps.